excludes/id.php

```diff
@@ -127,7 +138,7 @@
     'Tajikistan',
     'Tanzania',
     'Thailand',
-    'Timor-Leste',
+    'Timor - Leste',
```

Comment:
Invalid key name.

Correct is `Timor-Leste`

See: https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L545